user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md

```diff
@@ -32,115 +41,120 @@ To register a new Catalog item:
 
 1. Once all information has been filled in, click *Register*.
 
-   This will conclude the registration process for the Catalog item and return the unique identifier for the item in question. This identifier will allow you to register versions for the Catalog item using the [API](#using-the-catalog-api).
-
-> [!NOTE]
-> If the archive containing the new Catalog item version contains a readme.md file, the content of this file will be shown in the description tab for the Catalog item. Note that images are currently not yet supported in this readme file.<!-- RN 40190+40241 -->
+   This will conclude the registration process for the Catalog item and return the unique identifier for the item in question. This identifier will allow you to register versions for the Catalog item or update additional properties using the [Catalog API](#using-the-catalog-api).
 
 ## Using the Catalog API
 
-### Authentication
+With the Catalog API, you can both [register a Catalog item](#registering-a-catalog-item-with-the-api) and [register a new version of a Catalog item](#registering-a-new-version-with-the-api).
+
+>[!important]  
+The API calls are authenticated using [organization keys](xref:Managing_DCP_keys#organization-keys). The key must have the *Register catalog items permission* and needs to be added to the HTTP request in a header called **Ocp-Apim-Subscription-Key**.  
```

Comment:
Missing '>', the Important doesn't compile properly right now
![image](https://github.com/user-attachments/assets/716162d5-5d67-4972-9ec2-6af863e537da)

Reply:
prolly got fixed in between, looks ok now?

Reply:
ok